src/components/Tool/FixCommitmentInfo/Contact.test.tsx

```diff
@@ -3,159 +3,188 @@ import { ThemeProvider } from '@mui/material/styles';
 import userEvent from '@testing-library/user-event';
 import TestRouter from '__tests__/util/TestRouter';
 import TestWrapper from '__tests__/util/TestWrapper';
-import { render } from '__tests__/util/testingLibraryReactMock';
+import {
+  fireEvent,
+  render,
+  waitFor,
+} from '__tests__/util/testingLibraryReactMock';
+import { PledgeFrequencyEnum } from 'src/graphql/types.generated';
 import theme from '../../../theme';
 import Contact from './Contact';
 
-const testData = {
-  id: 'test123',
-  name: 'Test test',
+let testData = {
+  id: 'test 1',
+  name: 'Tester 1',
+  avatar: '',
   statusTitle: 'Partner - Financial',
-  statusValue: 'partner-financial',
+  statusValue: 'NEW_CONNECTION',
   frequencyTitle: 'Monthly',
-  frequencyValue: 'monthly',
+  frequencyValue: PledgeFrequencyEnum.Monthly,
   amount: 50,
-  amountCurrency: 'CAD',
+  amountCurrency: 'ARM',
+  donations: {
+    nodes: [
+      {
+        amount: {
+          amount: 175,
+          currency: 'USD',
+          conversionDate: '2019-10-15',
+          convertedCurrency: 'USD',
+        },
+      },
+    ],
+  },
 };
 
+let statuses = ['Partner - Financial', 'test_option_1'];
```

Comment:
Let's make this a prop that can be passed to `TestComponent` (possibly with a default value) instead of modifying its value partway through the tests.